packages/blitz/src/index.ts

```diff
@@ -4,6 +4,7 @@ export * from "@blitzjs/core/dynamic"
 export * from "@blitzjs/core/head"
 export * from "@blitzjs/core"
 export * from "@blitzjs/core/server"
+export * from "@blitzjs/display"
```

Comment:
Hey thanks @jmcelreavey!

So this actually doesn't expose the core logger.

1. What we need to expose is [`baseLogger`](https://github.com/blitz-js/blitz/blob/canary/nextjs/packages/next/server/lib/logging.ts#L31), although I think we should export it as `logger` instead of `baseLogger`.
2. And to export it, we should move that `logging.ts` file into this directory: https://github.com/blitz-js/blitz/tree/canary/nextjs/packages/next/stdlib-server
3. And then re-export `logger` from `next/stdlib-server/index.ts`
4. We already export `* from 'next/stdlib-server'` here in `blitz/src/index.ts`
5. But you need to add `logger` to this import rewrite list: https://github.com/blitz-js/blitz/blob/canary/packages/babel-preset/src/rewrite-imports.ts#L28-L38

Lastly, next time please don't remove the PR template from the description :) It has a critical checklist, namely ensure changes are covered by tests (this change doesn't need a new test) and opening a PR to the blitz docs repo to document any new features. And you are doing that, so a doc PR will be required.

Reply:
Apologies! Re-added the PR template and will update this PR accordingly. Thanks for all the helpful pointers :)

Reply:
@flybayer Are we sure we want to expose baseLogger? `log` within `@blitzjs/display` gives us a lot more functionality such as spinners and chalk.

Reply:
To start with, let's do just baseLogger. Users can always import and use chalk directly. I've found baseLogger is almost always what you want in app dev. Chalk is more useful when building CLIs.

We can always add more later if there is demand